services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -78,6 +78,9 @@ def _get_environment_variables(
         "S3_SECRET_KEY": r_clone_settings.R_CLONE_S3.S3_SECRET_KEY,
         "S3_SECURE": f"{r_clone_settings.R_CLONE_S3.S3_SECURE}",
         "SC_BOOT_MODE": f"{app_settings.DYNAMIC_SERVICES.DYNAMIC_SIDECAR.DYNAMIC_SIDECAR_SC_BOOT_MODE}",
+        "SSL_CERT_FILE": f"{app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME}"
```

Comment:
There is no need of the ``if`` statement since the original field is ``Optional[str]``, i.e. just
```python
"SSL_CERT_FILE": app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME
```

Reply:
@mrnicegyu11 NOTE that my example has no ``f"{ }"``. Initially you wrote:

```python
        "SSL_CERT_FILE": f"{app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME}"
        if app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME
        else None,
```
I suggested to simplify to
```python
"SSL_CERT_FILE": app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME
```
and you wrote
```python
"SSL_CERT_FILE": f"{app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME}"
```

First and second are equivalent. While the third will be wrong if ``app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME`` is ``None`` since it will write **the string** ``"None"``


PS: please do not forget answer and click resolve conversations before asking re-review. thx